ci/build_python.sh

```diff
@@ -25,4 +27,18 @@ RAPIDS_PACKAGE_VERSION=$(head -1 ./VERSION) rapids-conda-retry mambabuild \
   --channel "${CPP_CHANNEL}" \
   conda/recipes/pylibwholegraph
 
+sccache --show-adv-stats
+
+RAPIDS_PACKAGE_VERSION=$(head -1 ./VERSION) rapids-conda-retry mambabuild \
+  --no-test \
+  --channel "${CPP_CHANNEL}" \
+  --channel "${RAPIDS_CONDA_BLD_OUTPUT_DIR}" \
```

Comment:
Do we need `${RAPIDS_CONDA_BLD_OUTPUT_DIR}` here? What packages is it supplying? I don't see `pylibwholegraph` in the dependencies of [cugraph-pyg](https://github.com/rapidsai/cugraph-gnn/blob/branch-24.12/conda/recipes/cugraph-pyg/meta.yaml) or [cugraph-dgl](https://github.com/rapidsai/cugraph-gnn/blob/branch-24.12/conda/recipes/cugraph-dgl/meta.yaml).

Same for cugraph-dgl below.

Reply:
You're totally right. `pylibwholegraph` is a *testing* dependency of both `cugraph-pyg` and `cugraph-dgl`, but not a build-time dependency, so `--channel "${RAPIDS_CONDA_BLD_OUTPUT_DIR}"` is unnecessary. I just pushed https://github.com/rapidsai/cugraph-gnn/pull/59/commits/1202c36c0b27e342dc6f0a9ac95c01050e2064aa removing it.